_posts/2022-11-21-changelog-183.markdown

```diff
@@ -0,0 +1,92 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #183"
+date: 2022-11-21 13:00:00 +0300
+---
+
+
+## New Features
+
+* [#9694] Enable [derive](https://doc.rust-lang.org/reference/procedural-macros.html#derive-macros) procedural macro expansion in [nightly plugin builds](https://plugins.jetbrains.com/plugin/8182-rust/docs/rust-quick-start.html#install-nightly).
```

Comment:
Let's move it to the end of section since it's more technical change not fully related to stable release